device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -1232,7 +1232,7 @@ private TransportException singleReconnectAttempt()
     /**
      * Task for adding a packet back to the waiting queue. Used for delaying message retry
      */
-    public class MessageRetryRunnable implements Runnable
+    public static class MessageRetryRunnable implements Runnable
```

Comment:
although this class is public, there is no scenario in which users would initialize an instance of this class and pass it to the sdk